downloads/submodule-vs-subtree-cheat-sheet.md

```diff
@@ -216,13 +317,18 @@ Always show the submodule log when you diff:
 
     git config --global diff.submodule log
 
-Show a short summary of submodule changes in your status message:
+Show a short summary of submodule changes in your `git status` message:
+
+    git config --global status.submoduleSummary true
+
+Make `push` default to `--recurse-submodules=on-demand`:
+
+    git config --global push.recurseSubmodules on-demand
 
-    git config status.submoduleSummary true
+Make all commands (except `clone`) default to `--recurse-submodules` if they support the flag:
```

Comment:
```suggestion
Make all commands (except `clone`) default to `--recurse-submodules` if they support the flag (this works for `git pull` since Git 2.15):
```